### PR TITLE
Exclusive support for php 7 

### PR DESCRIPTION
**Pull request by hassansin**
* Dropped support for php < 7
* Updated dependencies to their latest
* Removed auto-generated docs

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,13 @@
 language: php
 php:
-  - '5.5'
-  - '5.6'
+  - '7.1'
   - '7.2'
-  - nightly
```
Comment (pkopac) [inline .travis.yml]: Why aren't we compatible with upcoming versions? Is it due to hard dependencies in libs? Can we at least have something like `7.* (latest)`?

**Comment by pkopac** [comment]
@hassansin oh, and you should explain what version of the library to use for which version of PHP in the README.

File: .travis.yml (hunk 1 above)
Comment (hassansin) [reply .travis.yml]: added `nightly` back

**Comment by pkopac** [comment]
@hassansin thanks. It seems we were way too inclusive, the end of life happened for the 5.* versions already at the beginning of the year: https://www.php.net/supported-versions.php

**Pull request merged**